walkthroughs/week-3-server/step-3-json-walkthrough.md

```diff
@@ -60,9 +59,9 @@ Here is an array that contains three objects:
 
 ```json
 [
-  {"name": "Ada", "birthYear": 1815},
-  {"name": "Grace", "birthYear": 1906},
-  {"name": "Alan", "birthYear": 1912}
+  { "name": "Ada", "birthYear": 1815 },
```

Comment:
For here an below, undo the extra spaces added?

Reply:
That is a file I was given by them to look at. I did not change it.

Reply:
That is a file I was given by them to look at. I did not change it.